README.md

```diff
@@ -122,51 +124,38 @@ To run tests, run the following command:
 
 ```sh
   gem install rspec
-  bundle exec rspec
+  rspec spec
 ```
 
-<!--
 ### Deployment
 
-You can deploy this project using:
+You can deploy this project using [render](https://render.com)
 
-<!--
-Example:
-
-```sh
-
-```
- -->
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- AUTHORS -->
 
 ## 👥 Authors <a name="authors"></a>
 
-👤 **Nickson Nowembabazi**
+> Mention all of the collaborators of this project.
 
-- GitHub: [@nowembabazi](https://github.com/Nowembabazi)
-- Twitter: [@nowembabazin](https://twitter.com/NowembabaziN)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/nowembabazi-nickson)
+👤 **Author1**
 
-👤 **Oluwatoyin Olaoye**
-
-- GitHub: [@AbayomiOlaoye](https://github.com/abayomiolaoye)
-- Twitter: [@Oluwatoyin](https://twitter.com/olaoyeelijah)
-- LinkedIn: [Oluwatoyin Olaoye](https://www.linkedin.com/in/oluwatoyinolaoye/)
+- GitHub: [@Nowembabazi](https://github.com/Nowembabazi)
+- Twitter: [@NowembabaziN](https://twitter.com/NowembabaziN)
+- LinkedIn: [nowembabazi-nickson](https://linkedin.com/in/nowembabazi-nickson)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- FUTURE FEATURES -->
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-We hope to implement these in the future in order to further boost interaction and increase use cases:
+> Describe 1 - 3 features you will add to the project.
 
```

Comment:
**[Optional]**

- [ ] It would be nice if you remove this section description to make your readme look clean and professional 😊 These section descriptions are just instructions for you when you will make your readme file and after making your readme file you are supposed to remove all of them because these are unnecessary for the viewers of your repository 🤔